app/services/data_migrations/backfill_sites_from_temp_sites.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrations
+  class BackfillSitesFromTempSites
+    TIMESTAMP = 20220620162528
+    MANUAL_RUN = false
+
+    def change
+      TempSite.find_each do |temp_site|
+        site = Site.create_or_find_by(
+          temp_site.attributes.except(%w[id created_at updated_at]),
```

Comment:
```suggestion
          temp_site.attributes.except('id', 'created_at', 'updated_at'),
```
otherwise those aren't ignored

Reply:
Doesn't this change upset rubocop?